lib/rabbitmq/cli/command_behaviour.ex

```diff
@@ -29,7 +29,9 @@ defmodule RabbitMQ.CLI.CommandBehaviour do
                       scopes: 0,
                       usage_additional: 0,
                       switches: 0,
-                      aliases: 0
+                      aliases: 0,
+                      required_rabbit_app_state: 2,
+                      validate_execution_environment: 2
```

Comment:
We have at least 17 commands, which do not care about rabbit app status:
- encode
- decode
- list_ciphers
- list_hashes 
- environment
- eval
- exec
- help
- hipe_compile
- stop
- shutdown
- cipher_suites
- erlang_cookie_hash
- erlang_version
- memory_breakdown
- server_version
- status

My point is that this behaviour is already a problem in this PR (handling `badrpc` errors) and will create more problems as more commands are added.